rackcity/models/asset.py

```diff
@@ -31,7 +31,8 @@ def get_assets_for_cp(change_plan=None):
     for assetCP in assetsCP:
         if (assetCP.related_asset) and (assetCP.related_asset) in assets:
             assets = assets.filter(~Q(id=assetCP.related_asset.id))
-    # TODO remove any Asset or AssetCP that has been decommissioned in this CP
+    if not show_decommissioned:
+        assetsCP = AssetCP.objects.filter(change_plan=change_plan, is_decommissioned=False)
```

Comment:
you can just apply the filter on the query set you already have
```
assetsCP = assetsCP.filter(is_decommissioned=False)
```
so you don't have to refilter on change_plan